lib/internal/Magento/Framework/Phrase/__function.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
Which rule was violated by `__.php` name?

Reply:
It was actually the function name itself `__()` which triggered the violation. As this was previously not occurring I assumed it was related to the file name. It turned out that wasn't the case and that __() was always in violation but making the changes brought this up.

If you have a strong preference I can change the filename __function.php back to __.php.

Reply:
Please do so.